docs/docs/stories.mdx

```diff
@@ -185,3 +187,56 @@ into a single file for evaluation. Read more about this format in [Testing Your
 This format is only used for testing and cannot be used for training.
 
 :::
+
+
+## End-to-end Training
+
+:::caution experimental feature
+End-to-end training is an experimental feature.
+We introduce experimental features to get feedback from our community, so we encourage you to try it out!
+However, the functionality might be changed or removed in the future.
+If you have feedback (positive or negative) please share it with us on the [Rasa Forum](https://forum.rasa.com).
+
+:::
+
+With end-to-end training, you do not have to deal with the specific
+intents of the messages that are extracted by the NLU pipeline
+or with separate `utter_` responses in the domain file.
+Instead, you can include the text of the user messages and/or bot responses directly in your stories.
+See the [training data format](./training-data-format.mdx#end-to-end-training)
+for detailed description of how to write end-to-end stories.
+
+You can mix training data in the end-to-end format with labeled training data which has 
+`intent`s and `action`s specified: Stories can have some steps defined by intents/actions
+and other steps defined directly by user or bot utterances.
+
+We call it end-to-end training because policies can consume and predict actual text.
+For end-to-end user inputs, intents classified by the NLU pipeline
+and extracted entities are ignored.
+
+
+Only [Rule Policy](./policies.mdx#rule-policy)
```

Comment:
should we mention that they should set `intent_tokenization_flag: True`?

Reply:
why? they don't have to use it. It is optional

Reply:
Ok - maybe my real question is: Why did we set this in the example bot?

Reply:
tokenizing names is an option that is nice to have for advanced users if they create consistent action names, it is not required to train e2e model

it is an optional configuration decision that might improve bot performance